### ISSUE TITLE
aria-label missing from "More" toolbar button

### ISSUE THREAD

**Issue by rmmayo** [nhn/tui.editor#2784]
The "More" button is missing the `aria-label` attribute which violates the `button-rule` accessibility rule.  Add the `arial-label="More"` attribute would resolve the issue.

Please see the following for more info: https://dequeuniversity.com/rules/axe/4.4/button-name?application=axeAPI

![image](https://user-images.githubusercontent.com/47030042/195426042-70eee4de-e708-4b47-b0e0-c7311f37fadd.png)
![image](https://user-images.githubusercontent.com/47030042/195426402-1050f1dd-cc0f-4723-9454-db2c450f365a.png)

**Comment by jwlee1108** [comment]
Thank you for the issue. we'll check it later.
